src/main/java/frc/robot/subsystems/InterfaceSubsystem.java

```diff
@@ -35,20 +42,150 @@ public InterfaceSubsystem(
+                        FieldConstants.getSetPoint(Place.G_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.G_TREE).getRotation());
+            break;
+          case "h":
+            targetTransform =
+                new Transform2d(
+                    new Translation2d(
+                        FieldConstants.getSetPoint(Place.H_TREE).getX(),
+                        FieldConstants.getSetPoint(Place.H_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.H_TREE).getRotation());
+            break;
+          case "i":
+            targetTransform =
+                new Transform2d(
+                    new Translation2d(
+                        FieldConstants.getSetPoint(Place.I_TREE).getX(),
+                        FieldConstants.getSetPoint(Place.I_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.I_TREE).getRotation());
+            break;
+          case "j":
+            targetTransform =
+                new Transform2d(
+                    new Translation2d(
+                        FieldConstants.getSetPoint(Place.J_TREE).getX(),
+                        FieldConstants.getSetPoint(Place.J_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.J_TREE).getRotation());
+            break;
+          case "k":
+            targetTransform =
+                new Transform2d(
+                    new Translation2d(
+                        FieldConstants.getSetPoint(Place.K_TREE).getX(),
+                        FieldConstants.getSetPoint(Place.K_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.K_TREE).getRotation());
+            break;
+          case "l":
+            targetTransform =
+                new Transform2d(
+                    new Translation2d(
+                        FieldConstants.getSetPoint(Place.L_TREE).getX(),
+                        FieldConstants.getSetPoint(Place.L_TREE).getY()),
+                    FieldConstants.getSetPoint(Place.L_TREE).getRotation());
+            break;
         }
+        DriveCommands.goToTransformWithPathFinder(drive, targetTransform)
+            .andThen(DriveCommands.goToTransform(drive, targetTransform))
+            .beforeStarting(
+                () -> {
+                  DriveCommands.goToTransform(drive, targetTransform).cancel();
+                  DriveCommands.goToTransformWithPathFinder(drive, targetTransform).cancel();
+                });
         break;
       case CORAL:
-        // Drive commands to drive to coral station
+        targetTransform =
+            new Transform2d(
+                new Translation2d(
+                    FieldConstants.getSetPoint(Place.LEFT_CORAL_STATION).getX(),
+                    FieldConstants.getSetPoint(Place.LEFT_CORAL_STATION).getY()),
+                FieldConstants.getSetPoint(Place.LEFT_CORAL_STATION).getRotation());
+        DriveCommands.goToTransformWithPathFinder(drive, targetTransform)
```

Comment:
🎨 It's still repeated in these other cases, so just make a private method in this class and call it all the places you need it and these commands are written (in code) once.